app/src/main/java/com/github/se/wanderpals/model/firestoreData/FirestoreSuggestion.kt

```diff
@@ -30,7 +32,9 @@ data class FirestoreSuggestion(
     val stop: FirestoreStop = FirestoreStop(), // Using Firestore-compatible Stop object
     val comments: List<FirestoreComment> =
         emptyList(), // Using Firestore-compatible Comment objects
-    val userLikes: List<String> = emptyList()
+    val userLikes: List<String> = emptyList(),
+    val stopStatus: String =
```

Comment:
like said previously, not required. Should use the text of suggestion to mark accepted suggestion like discussed.